rag/llm/rerank_model.py

```
@@ -298,6 +299,105 @@ def _compute_rank(self, query: str, texts: List) -> Tuple[np.ndarray, int]:
+                    "type": "BEDROCK_RERANKING_MODEL",
+                    "bedrockRerankingConfiguration": {
+                        "numberOfResults": len(batch),
+                        "modelConfiguration": {"modelArn": self.model_arn},
+                    },
+                },
+            )
+            try:
+                for d in res.get("results", []):
+                    rank[offset + d["index"]] = d["relevanceScore"]
+                    result_count += 1
+            except Exception as _e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)